app/src/androidTest/java/org/mozilla/fenix/perf/StartupExcessiveResourceUseTest.kt

```diff
@@ -99,6 +107,7 @@ class StartupExcessiveResourceUseTest {
             EXPECTED_RECYCLER_VIEW_CONSTRAINT_LAYOUT_CHILDREN,
             actualRecyclerViewConstraintLayoutChildren
         )
+        assertEquals(failureMsgNumberOfInflation, EXPECTED_COMPONENT_INIT_COUNT, actualNumberOfInflations)
```

Comment:
We're using the wrong expected value here (maybe that's why the UI test failed?)